### ISSUE TITLE
Editing of phones is partially implemented

### ISSUE THREAD

**Issue by darko-mijic** [jagi/meteor-astronomy-examples#7]
Hi, I can se that editing the list of phones is partially implemented. Templates are there but there is no logic. Are you planing to complete the example?

**Comment by lukejagodzinski** [comment]
Yes, I'm planing to complete it after I've implement the `forms` package. Right now it's possible to implement it in many different ways. And I want to come up with the best one solution (using forms package) that I will use in the example.

**Comment by darko-mijic** [comment]
Yes, I know there are many different ways for implementing it. I tried quite a few. I was hoping to see Your thinking on that subject.

Do You have any estimates for the release of forms package? Do You have a preview of that project and do You need any help? I know that You are building much more flexible solution than autoforms, something more like useful:forms, right?  

I am considering the best architecture for new Meteor project and my thinking is that Astronomy and ODM approach is much better solution then using SimpeSchema, and many other packages... Even thought Astronomy is so new.

**Comment by lukejagodzinski** [comment]
I hope that it will take around one month but I don't want to provide any date because I'm in the early stage and it may happen that I won't meet a goal.

I will definitely need a help with testing it. I will let know here on github and gitter when the testing phase is on.

Yes, I'm building something much more flexible. In deed, I will be borrowing some ideas from useful:forms. I'm in contact with developers from Useful.io and other packages creators. I hope we will come to the best possible solution for the forms problem.

I'm glad to hear that :). There are more and more developers that want to use Astronomy in production. It makes me proud but with a great power comes a great responsibility :). It would be nice if you could share your opinion [here](https://github.com/meteor/guide/issues/10). MDG is planing to release an official guide and they need to choose ORM/ODM tool they will be describing.

I would recommend you to join [gitter](https://gitter.im/jagi/meteor-astronomy) group, where I will inform about new versions and needed help.

Thanks!

**Comment by darko-mijic** [comment]
Can't wait for testing phase. :)

I was planning to comment  [Collections and models: ideas](https://github.com/meteor/guide/issues/10) after work, I was reading it this morning.

**Comment by lukejagodzinski** [comment]
Ok good to hear :). Will let know about testing phase. Stay tuned :)

**Comment by darko-mijic** [comment]
I have implemented the simplest possible solution for editing the list of phones in your example app. 
I am storing the complete contact that is being edited or created in a Reactive Var and updating it when 'change input' event fires. It is [here](https://github.com/darko-mijic/meteor-astronomy-examples).

**Comment by lukejagodzinski** [comment]
Ok thanks! I will take a look at it today.
